### PR TITLE
[snyk] Add snyk docker image scanning github action

### PR DESCRIPTION
**Pull request by jsugarman**
#### What
Add a github action that can scan and monitor latest docker image on a schedule

#### Why
github actions are free and allow us to handle success/failure 
with a conditional slack notification more easily.

### How
This action builds an image and scans it.  
Success/failures will alert will to laa-cccd-alerts channel. We can easily changes this

### CODE REVIEW

File: .github/workflows/docker_image_scan.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        echo ${{ steps.image-scan.outcome }}
+    - name: Slack notify failure
+      if: ${{ steps.image-scan.outcome == 'failure' }}
+      uses: rtCamp/action-slack-notify@v2
+      env:
+        SLACK_USERNAME: Snyk docker image scan
+        SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+        SLACK_CHANNEL: laa-cccd-alerts
+        SLACK_ICON_EMOJI: ':snyk:'
+        SLACK_COLOR: '#fc0303'
```
Comment (jrmhaig) [inline .github/workflows/docker_image_scan.yml:42]: (nit) If there is a 'success' colour (below) is there also a named colour for failure?

**Review by jrmhaig — APPROVED** [review]
:+1:

File: .github/workflows/docker_image_scan.yml (hunk 1 above)
Comment (jsugarman) [reply .github/workflows/docker_image_scan.yml:42]: I think it was purple or I could not find one and purple was the default. I forget now. We can tweak on a separate branch i'd say

Comment (jsugarman) [reply .github/workflows/docker_image_scan.yml:42]: >        SLACK_COLOR: ${{ job.status }} # or a specific color like 'good' or '#ff00ff'
https://github.com/rtCamp/action-slack-notify

**Pull request merged**